collector/falcon.go

```diff
@@ -0,0 +1,17 @@
+package collector
+
+import falcon "git.n.xiaomi.com/falcon-sdk/goperfcounter"
+
+type FalconCounter struct {
+	name string
+}
+
+func (f *FalconCounter) Incr() {
+	falcon.SetCounterCount(f.name, 1)
+}
+
+func registerFalconCounter(counterName string) *FalconCounter {
```

Comment:
Certainly not true. Theorically you can specify tags for every single counter separately. But it could be limited according to the falcon sdk you use.